### DISCUSSION TITLE
This plugin slows down Homebridge message

### DISCUSSION THREAD

**Discussion by drlkp** [plasticrake/homebridge-tplink-smarthome#281] [General]
Hi,
how to solve? 
[homebridge-tp-link-tapo] This plugin slows down Homebridge. The read handler for the characteristic 'On' was slow to respond! See https://homebridge.io/w/JtMGR for more info.

**Comment by plasticrake** [comment]
Wrong plugin. I think you meant: https://github.com/RaresAil/homebridge-tp-link-tapo

**Reply by drlkp** [reply]
yes you are right
